07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,57 @@
+h1 {
+    color: #313131;
+    padding-left: 234px;
+}
+.header {
+    background-color: #313131;
+    font-family: sans-serif;
+    height: 50px;
+    padding-left: 94px;
+}
+.header a {
+    color: white;
+    text-decoration: none;
+    font-family: sans-serif;
+    font-size: 14px;
+    padding-top: 20px;
+    padding-left: 30px;
+}
+nav {
+    margin-left: 124px;
+    line-height: 50px;
+}
+#HR {
+    margin-top: 10px;
+    margin-left: 234px;
+}
+.left-box {
+    font-family: sans-serif;
+    font-size: 14px;
+    color: white;
+    background-color: #55a747;
+    width: 40%;
+    height: 200px;
+    float: left;
+}
+.right-box {
+    font-family: sans-serif;
+    font-size: 14px;
+    color: white;
+    background-color: #313131;
+    width: 40%;
+    height: 200px;
+    float: left;
+}
+h2, p, a {
```

Comment:
If you can get those link boxes to align right with the text, I'll bump you to 100%.